script.js

```diff
@@ -10,16 +10,38 @@ const upgradeButton = document.getElementById("upgrade-button");
 const powerUpButton = document.getElementById("powerup-button");
 const gameContainer = document.getElementById("game-container");
 
+// Calculate the value of a click at this current moment
+function getClickValue(){
+  // This can get fancier if the powerUp ever gets upgrades :D 
+  const powerupMultiplier = powerUpActive ? 2 : 1;
```

Comment:
FYI this is shorthand syntax. 
Says "If powerUpActive is true, then set powerupMultiplier to 2, otherwise set it to 1."